sdk/keyvault/keyvault-keys/test/list.test.ts

```diff
@@ -201,7 +201,7 @@ describe.only("Keys client - list keys in various ways", () => {
     for await (const page of client.listDeletedKeys().byPage()) {
       for (const deletedKey of page) {
         // The vault might contain more keys than the ones we inserted.
-        if (!keyNames.includes(deletedKey.properties.name)) continue;
+        if (!keyNames.includes(deletedKey.name)) continue;
```

Comment:
Did you make similar changes in your samples? We want to highlight that users can get id and name directly from the key instead of key.properties (though, that would also work).

Reply:
The changes don't read properties.name